docs/sources/installation/requirements.md

```diff
@@ -23,31 +25,32 @@ The following operating systems are supported for Grafana installation:
 - [macOS](/installation/mac)
 - [Windows](/installation/windows)
 
-## Unsupported operating systems
+While you can install Grafana on other operating systems, we do not recommend doing so and do not provide support for that use case.
 
-Installation of Grafana on other operating systems is possible, but not supported. Please see the [building from source](/project/building_from_source/#building-grafana-from-source) guide for more information.
+## Hardware recommendations
```

Comment:
This is super wishy-washy for a requirements page. Are there hard and fast requirements? Can we be a little more specific on the recommendations for hardware?

Reply:
Last time I created this requirement page it was super hard to pinpoint the exact requirements so that's why it looks like it does and why this issue, #19101, is still open. Please remember we didn't have a requirements page a couple of months ago :)

Reply:
That is a "no," then. Okeydokey.

Reply:
Well, for now if we don't have anything better to add. But we should be able to add something better here. We can maybe check Hosted Grafana statistics to get some insights.

Reply:
I will keep my eyes open. Creating/adding Cloud Grafana and Grafana Enterprise documentation is high on my To Do list.